documentation/modules/security/ref-certificates-and-secrets.adoc

```diff
@@ -5,41 +5,75 @@
 Secrets are used for establishing TLS encrypted connections between Kafka brokers, and between brokers and clients.
 They are also used for mutual TLS authentication.
 
-* A _Cluster Secret_ contains a cluster CA certificate to sign Kafka broker certificates, and is used by a connecting client to establish a TLS encrypted connection with the Kafka cluster to validate broker identity.
-* A _Client Secret_ contains a client CA certificate for a user to sign its own client certificate to allow mutual authentication against the Kafka cluster.
-The broker validates the client identity through the client CA certificate itself.
-* A _User Secret_ contains a private key and certificate, which are generated and signed by the client CA certificate when a new user is created.
-The key and certificate are used for authentication and authorization when accessing the cluster.
+Cluster secret:: A cluster secret contains the _cluster CA_ to sign Kafka broker certificates.
+Connecting clients use the certificate to establish a TLS encrypted connection with a Kafka cluster. The certificate verifies broker identity.
+Client secret:: A client secret contains the _clients CA_ for a user to sign its own client certificate.
+This allows mutual authentication against the Kafka cluster. The broker validates a client's identity through the certificate.
+User secret:: A user secret contains a private key and certificate. The secret is created and signed by the clients CA when a new user is created. The key and certificate are used to authenticate and authorize the user when accessing the cluster.
 
-Secrets provide private keys and certificates in PEM and PKCS #12 formats.
-Using private keys and certificates in PEM format means that users have to get them from the Secrets, and generate a corresponding truststore (or keystore) to use in their Java applications.
-PKCS #12 storage provides a truststore (or keystore) that can be used directly.
-
-All keys are 2048 bits in size.
+== Secrets in PEM and PKCS #12 formats
 
-== PKCS #12 storage
+Secrets provide private keys and certificates in PEM and PKCS #12 formats.
+Using private keys and certificates in PEM format means that users have to get them from the secrets, and generate a corresponding truststore or keystore to use in their Java applications.
+PKCS #12 storage provides a truststore or keystore that can be used directly.
 
 PKCS #12 defines an archive file format (`.p12`) for storing cryptography objects into a single file with password protection.
 You can use PKCS #12 to manage certificates and keys in one place.
 
-Each Secret contains fields specific to PKCS #12.
+Each secret contains fields specific to PKCS #12.
 
 * The `.p12` field contains the certificates and keys.
 * The `.password` field is the password that protects the archive.
 
-== Cluster CA Secrets
+All keys are 2048 bits in size and are valid for 365 days from initial generation.
```

Comment:
The 365 is by default but can be changed. Should we mention it?

Reply:
added link to section that describes changing period